src/gui/utils.cpp

```diff
@@ -146,7 +146,22 @@ void Utils::Gui::openPath(const Path &path)
     const QUrl url = path.data().startsWith(u"//")
         ? QUrl(u"file:" + path.data())
         : QUrl::fromLocalFile(path.data());
+
+#ifdef Q_OS_WIN
+    auto *thread = QThread::create([path]()
+    {
+        if (SUCCEEDED(::CoInitializeEx(NULL, COINIT_APARTMENTTHREADED | COINIT_DISABLE_OLE1DDE)))
+        {
+            ShellExecute(nullptr, nullptr, path.toString().toStdWString().c_str(), nullptr, nullptr, SW_SHOWNORMAL);
```

Comment:
I would also extract it to a variable to avoid potential lifetime issues:
```suggestion
            const std::wstring pathWStr = path.toString().toStdWString();
            ::ShellExecuteW(nullptr, nullptr, pathWStr.c_str(), nullptr, nullptr, SW_SHOWNORMAL);
```

Reply:
> I would also extract it to a variable to avoid potential lifetime issues

There should not be issues since such temporaries live until end of the function. But at least it can improve the readability.